pkg/agent/openflow/client.go

```diff
@@ -97,14 +96,13 @@ type Client interface {
 	// InstallEndpointFlows.
 	UninstallEndpointFlows(protocol binding.Protocol, endpoints []proxy.Endpoint) error
 
-	// InstallServiceFlows installs flows for accessing Service NodePort, LoadBalancer and ClusterIP. It installs the
-	// flow that uses the group/bucket to do service LB. If the affinityTimeout is not zero, it also installs the flow
-	// which has a learn action to maintain the LB decision. The group with the groupID must be installed before,
-	// otherwise the installation will fail.
-	// nodeLocalExternal represents if the externalTrafficPolicy is Local or not. This field is meaningful only when
-	// the svcType is NodePort or LoadBalancer.
-	// nested represents if the Service has the Endpoints which is other Service's ClusterIP.
-	InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16, nodeLocalExternal bool, svcType v1.ServiceType, nested bool) error
+	// InstallServiceFlows installs flows for accessing Service NodePort, LoadBalancer, ExternalIP and ClusterIP. It
+	// installs the flow that uses the group/bucket to do Service LB. If the affinityTimeout is not zero, it also
+	// installs the flow which has a learn action to maintain the LB decision. The group with the groupID must be
+	// installed before, otherwise the installation will fail.
+	// externalAddress indicates that whether the Service is externally accessible, like NodePort, LoadBalancer and ExternalIP.
+	// nested indicates that whether the Service are backed by Endpoints of other Services.
```

Comment:
It sounds the Service and other Services are sharing Endpoints, but we actually mean the Service's Endpoint is other Service's ClusterIPs, right?
Nested Service's ClusterIP -> Nested Service's Endpoint (Other Service's ClusterIP) -> Other Service's Endpoint

How about "nested indiciates the whether the Service are backed by Service IPs of other Services"? @jianjuns @hongliangl

Reply:
I think it is more readable by explaining `be backed by Service IPs of other Services`.